openpgp-dsm/src/lib.rs

```diff
@@ -616,20 +616,32 @@ impl KeyMetadata {
     }
 }
 
+pub enum KeyFlagsArg {
+    C,
+    CS,
+    S,
+    EtEr,
+}
+
 /// Generates an OpenPGP key with secrets stored in DSM. At the OpenPGP
 /// level, this method produces a key consisting of
 ///
-/// - A primary certifying key (flag C)
+/// - A primary certifying key (flag C) or primary signing key (flags C + S),
 /// - A signing subkey (flag S),
```

Comment:
Should we edit the doccomments, since, based on the values in `KeyFlagsArg`, we might not necessarily be creating 3 keys?

Reply:
resolving